docs/reference/api/pages.md

```diff
@@ -202,7 +202,7 @@ The successful GET request returns the matching document. See the [page document
 
 | Property | Type | Description |
 |----------|------|-------------|
-|`_targetId` | String | The `_id` of an existing page to use as a target when inserting the new page|
+|`_targetId` | String | The `_id` of an existing page to use as a target when inserting the new page. `_home` and `_trash` are special values for the home page and [trashed section](#moving-pages-to-the-trash), respectively. |
```

Comment:
How about: "`_home` and `_trash` are convenient shortcuts to save the trouble of looking up the `_id` of the home page and the main trash page, respectively"

I don't want it to sound like they are mandatory, you could use those _ids